### ISSUE TITLE
Add support for gzip compression

### ISSUE THREAD

**Issue by chrisdavenport** [joomla-projects/webservices#11]
If the requests includes 'gzip' in an Accept-Encoding header then the response should be gzipped.

**Comment by wilsonge** [comment]
Something that should probably be done at the application level - there's already base support for gzip in JApplicationWeb. We should turn it into a method to set gzip to true/false and then call that I think

**Comment by mbabker** [comment]
It's based on config already, you don't need a getter or setter around that.

**Comment by chrisdavenport** [comment]
Right.  So this can be fixed when the new entry point code is written.

**Comment by wilsonge** [comment]
Yup :)
